python/src/deltachat/account.py

```diff
@@ -62,12 +62,15 @@ class Account(object):
 
     MissingCredentials = MissingCredentials
 
-    def __init__(self, db_path, os_name=None, logging=True) -> None:
+    def __init__(self, db_path, os_name=None, logging=True, closed=False) -> None:
         """initialize account object.
 
         :param db_path: a path to the account database. The database
                         will be created if it doesn't exist.
-        :param os_name: this will be put to the X-Mailer header in outgoing messages
+        :param os_name: [Deprecated] this will be put to the X-Mailer header in outgoing messages
```

Comment:
it's just deprecated, there is no X-Mailer header anymore

Reply:
done